src/main/java/frc/robot/subsystems/arm/commands/SetArmsPosition.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+import         edu.wpi.first.math.MathUtil;
+import frc.robot.subsystems.arm.ArmConstants;
+
+public class SetArmsPosition extends CommandBase {
+    private final Arm arm;
+    private final Translation2d position;
+
+    public SetArmsPosition(Translation2d position) {
+        this.arm = Arm.getInstance();
+        this.position = position;
+        addRequirements(arm);
+    }
+
+    @Override
+    public void initialize() {
+        arm.setEndPosition(position);
+    }
+
+    @Override
+    public boolean isFinished() {
+       return MathUtil.applyDeadband(position.getNorm(), ArmConstants.SETPOINT_DEADBAND) == arm.getEndPosition().getNorm();
```

Comment:
The norm is the magnitude of the position, which is not what you want to check, I think it will be easier to just check k the x component and the y component

Reply:
So should I use the applyDeadband twice? once for the x component and once for the y

Reply:
Unfortunately yes